### PR TITLE
Remove storybook

### PR DESCRIPTION
**Pull request by krzysztofzuraw**
## What type of PR is this?
- [ ] 💅 Refactor
- [ ] 🌟 Feature
- [ ] 🔥 Bug Fix
- [ ] 🔩 Maintenance
- [ ] 🛠 Workflow CI/CD changes

## Related Issues or Documents

- closes #

## Have you written tests?
- [ ] Yes!
- [ ] No... here is why: _Writing tests are mandatory, please replace this text with why test are not included in this PR_

### CODE REVIEW

**Review by magul — COMMENTED** [review]
~~We also need to clean up our inrastructure,  can somebody retrieve:~~ 
* ~~AWS_ACCESS_KEY_ID~~
* ~~AWS_TEST_DEPLOYMENT_BUCKET~~
* ~~AWS_TEST_CF_DIST_ID~~

~~and let infrastructure guild know (I'm pretty confident we know what these values are, but let double-check).~~

EDIT: Ok, find them. This wasn't what I expected. Will create infrastructure ticket to clean leftovers we have in our infra.

**Comment by krzysztofzuraw** [comment]
@magul we still need those variables - I removed only job for storybook. Dashboard build & deployment is still in place 😄 
CleanShot 2024-03-18 at 12 56 21@2x

**Review by poulch — DISMISSED** [review]

**Review by NyanKiyoshi — APPROVED** [review]

**Review by poulch — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -318,7 +312,9 @@
     "dev": "vite --host",
     "build": "cross-env NODE_OPTIONS=--max-old-space-size=8192 vite build",
     "preview": "vite preview",
-    "build-storybook": "cross-env NODE_OPTIONS=--max-old-space-size=6144 storybook build -o build/storybook",
+    "sentry:inject": "sentry-cli sourcemaps inject build/dashboard",
+    "sentry:upload": "sentry-cli sourcemaps upload build/dashboard",
+    "sentry:clean": "node scripts/removeSourcemaps.js",
```
Comment (NyanKiyoshi) [inline package.json:317]: Unrelated changes, are those accidental?

Comment (poulch) [reply package.json:317]: Good catch @NyanKiyoshi, we should delete them

Comment (krzysztofzuraw) [reply package.json]: Removed :bo

**Review by NyanKiyoshi — APPROVED** [review]

**Review by poulch — APPROVED** [review]
